cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -12,6 +12,12 @@ metadata:
 spec:
   # The failure domain will spread the replicas of the data across different failure zones
   failureDomain: host
+  # Ceph CRUSH root location of the rule
```

Comment:
How about adding these comments after the `replicated` section?

Reply:
Like so?